cypress/common/if-exists.js

```diff
@@ -1,15 +1,25 @@
 /**
- * Executes the callback only if the element exists on the page.
+ * Executes the callback only if the program year selection element exists on the page.
  * 
- * @callback ifExistsCallback
+ * @callback ifProgramYearExistsCallback
  * 
- * @param {string} element Selector string for an element on the page.
- * @param {ifExistsCallback} callback Callback function to be executed only if the element exists.
+ * @param {ifProgramYearExistsCallback} callback Callback function to be executed only if the element exists.
 */
-export function ifExists(element, callback) {
-  cy.get('body').then((body) => {
-    if (body.find(element).length > 0) {
-        callback();
-    }
-  });
-};
+export function ifProgramYearFieldExists(callback) {
+  if (multipleProgramYears()) {
+      callback();
+  };
+
+  /** 
+   * Return true or false to determine if applicant can select multiple program years
+   * i.e., if test is running between 10/01 and 10/14
+  */
+   function multipleProgramYears() {
+      const todaysDate = new Date();
+      if (todaysDate.getMonth() + 1 == 10 && todaysDate.getDate() <= 14) {
+          return true;
+      } else {
+          return false;
+      }
```

Comment:
```suggestion
      return todaysDate.getMonth() + 1 == 10 && todaysDate.getDate() <= 14;
```
suggestion (non-blocking): You can also just return the condition itself which is already giving you the boolean value.